pyproject.toml

```diff
@@ -176,11 +176,16 @@ select = [
     "C4",  # flake8-comprehensions
     "BLE",  # flake8-blind-except
     "UP",  # pyupgrade
-    "RUF100",  # Report unused noqa directives
+    #"RUF100",  # Report unused noqa directives
+    #"RUF",  # Report unused noqa directives
```

Comment:
I meant to activate this. Any issues with it? We should otherwise leave the RUF100 active.

Reply:
Remove RUF line and activate RUF100 again.